spec/teaspoon_env.rb

```diff
@@ -0,0 +1,16 @@
+ENV['TEASPOON_RAILS_ENV'] = File.expand_path('../dummy/config/environment', __FILE__)
+
+Teaspoon.configure do |config|
+  config.mount_at = '/teaspoon'
+  config.root = UiComponents::Engine.root
+  config.asset_paths = ['spec/javascripts', 'spec/javascripts/stylesheets']
+  config.fixture_paths = ['spec/javascripts/fixtures']
+
+  config.suite do |suite|
+    suite.use_framework :jasmine, '2.2.0'
```

Comment:
The config was generated like this, but it does seem to use the latest version if we don't specify.  I updated it.  The teaspoon gem doesn't appear to support `2.3` yet, so that's why it was at `2.2`.